.github/workflows/build_rust_docs.yaml

```diff
@@ -39,12 +39,11 @@ jobs:
           docker rmi $(docker image ls --all --quiet)
           df --human-readable
 
-      # Build Docker image, caching from the latest version from the remote repository.
-      - name: Docker build
-        timeout-minutes: 30
+      - name: Docker pull
+        timeout-minutes: 10
         run: |
-          docker pull gcr.io/oak-ci/oak:latest
-          docker build --pull --cache-from=gcr.io/oak-ci/oak:latest --tag=gcr.io/oak-ci/oak:latest .
+          ./scripts/docker_pull
+          df --human-readable
```

Comment:
Why do we run this after running ./scripts/docker_pull (also ff)?

Reply:
Yes, the `df` command. I see, I was just wondering if the `df` should go into docker_pull script, because they appear together often. But probably not in that case.